src/olympia/files/management/commands/extract_optional_permissions.py

```diff
@@ -18,7 +18,8 @@ def handle(self, *args, **options):
         ).order_by('pk')
         pks = files.values_list('pk', flat=True)
 
-        log.info('Using %s file pks to extract permissions' % pks.count())
+        log.info('Using %s file pks to extract permissions, max pk: %s' %
+                 (pks.count(), pks[len(pks) - 1]))
```

Comment:
`pks[-1]` is the same as `pks[len(pks) - 1]`

Reply:
I tried that. I got "Negative indexing is not supported" :-(

Reply:
ah, must be a queryset thing.  I feel `len(pks)` should be `pks.count()` again though, or both `len` - seems inconsistent to have the same thing two different ways in one statement.